spec/requests/citizens/identify_types_of_incomes_spec.rb

```diff
@@ -69,6 +77,33 @@
       end
     end
 
+    context 'when "none selected" has been selected' do
+      let(:params) { { none_selected: 'true' } }
+
+      it 'does not add transaction types to the application' do
+        expect { subject }.not_to change { LegalAidApplicationTransactionType.count }
+      end
+
+      it 'redirects to the next step' do
+        expect(subject).to redirect_to(flow_forward_path)
+      end
+
+      context 'and application has transactions' do
+        let(:legal_aid_application) do
+          create :legal_aid_application, :with_applicant, transaction_types: income_types
+        end
+
+        it 'does not add transaction types to the application' do
```

Comment:
```suggestion
        it 'removes transaction types from the application' do
```
Would this make more sense?